src/App.tsx

```diff
@@ -8,53 +8,110 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { useEffect, useState } from 'react';
+import { getUserPosts, getUsers } from './client/clientMethods';
+import { User } from './types/User';
+import { Post } from './types/Post';
 
-export const App = () => (
-  <main className="section">
-    <div className="container">
-      <div className="tile is-ancestor">
-        <div className="tile is-parent">
-          <div className="tile is-child box is-success">
-            <div className="block">
-              <UserSelector />
-            </div>
+export const App = () => {
+  const [isError, setIsError] = useState<boolean>(false);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isError, setIsError] = useState(false);
  const [isLoading, setIsLoading] = useState(false);
```
it is better not to specify simple types